source/contributing/doing-a-release.rst

```diff
@@ -228,41 +158,34 @@ If needed, merge the changes into the develop branch.
     cd /home/frs/project/g/ge/geonetwork/GeoNetwork_opensource
     # or for RC release
     cd /home/frs/project/g/ge/geonetwork/GeoNetwork_unstable_development_versions/
-    mkdir v3.6.0
-    cd v3.6.0
-    put docs/changes3.6.0-0.txt
+    mkdir 3.0.0
```

Comment:
Is this just an example or the actual path?

Reply:
an example. The path is https://sourceforge.net/projects/geonetwork/files/GeoNetwork_opensource/ 

Not sure we can use the env variable from the FTP command line ?